# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnome-system-tools/crack-missing-header.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gnome-system-tools-0.27.0/src/users/passwd.c.orig	2003-10-01 02:42:42.000000000 +0200
+++ gnome-system-tools-0.27.0/src/users/passwd.c	2003-10-01 02:42:24.000000000 +0200
@@ -33,7 +33,7 @@
 #include <unistd.h>
 
 #ifdef HAVE_LIBCRACK
-#include <crack.h>
+const char *FascistCheck (const char *, const char *);
 #endif
 
 #ifdef HAVE_CRYPT_H
